eZ/Publish/Core/Repository/URLAliasService.php

```diff
@@ -73,12 +74,17 @@ public function __construct(RepositoryInterface $repository, Handler $urlAliasHa
      * @param string $languageCode the languageCode for which this alias is valid
      * @param bool $alwaysAvailable
      *
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException if the user is not allowed to create url alias
```

Comment:
Docblock for `\eZ\Publish\API\Repository\URLAliasService::createUrlAlias` should be updated (also for  `createGlobalUrlAlias` and `removeAliases` methods)